Makefile

```diff
@@ -0,0 +1,17 @@
+all: push
+
+# 0.0 shouldn't clobber any release builds
+TAG = 0.0
+PREFIX = kobolog/gorb
+
+binary: main.go
+	CGO_ENABLED=0 GOOS=linux godep go build -a -installsuffix cgo -ldflags '-w' -o docker/gorb ./main.go ./http.go
```

Comment:
> Why -installsuffix cgo

Removed.

> Also, I think it will build just fine without listing the source files in the end.

Without ./http.go I see this error
````
old-mbp:gorb aledbf$ make
CGO_ENABLED=0 GOOS=linux godep go build -a -ldflags '-w' -o docker/gorb ./main.go
# command-line-arguments
./main.go:74: undefined: serviceCreateHandler
./main.go:75: undefined: backendCreateHandler
./main.go:76: undefined: backendUpdateHandler
./main.go:77: undefined: serviceRemoveHandler
./main.go:78: undefined: backendRemoveHandler
./main.go:79: undefined: serviceStatusHandler
./main.go:80: undefined: backendStatusHandler
godep: go exit status 2
make: *** [binary] Error 1
```